### DISCUSSION TITLE
Root composer.json requires bref/bref ^2.0, found bref/bref

### DISCUSSION THREAD

**Discussion by danilorpneves** [dedoc/scramble#263] [General]
I was unable to install and the following message appeared.

"
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires bref/bref ^2.0, found bref/bref[2.0.0-beta1, ..., v2.x-dev] but the package is fixed to 1.7.5 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.
"

It seems that the package is not compatible with the new version of Bref, is there any alternative?

**Comment by romalytvynenko** [comment]
Should be ok in the newest version
